### ISSUE TITLE
How to get float type output data from "dummy-quantization" model?

### ISSUE THREAD

**Issue by raninbowlalala** [tensorflow/tensorflow#20726]
### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: No
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Linux Ubuntu 16.04
- **TensorFlow installed from (source or binary)**:binary
- **TensorFlow version (use command below)**:1.8.0
- **Python version**: 2.7.3
- **Bazel version (if compiling from source)**:0.12.0
- **GCC/Compiler version (if compiling from source)**: c++11
- **CUDA/cuDNN version**:7.5.18
- **GPU model and memory**:TITAN,12GB
- **Exact command to reproduce**:N/A

### Describe the problem
I use "dummy-quantization" to generate the quantization type model of deeplabv3(mobilenetv2), the command like below:
`
 --input_format=TENSORFLOW_GRAPHDEF \
  --output_format=TFLITE \
  --inference_type=QUANTIZED_UINT8 \
  --input_shape=1,513,513,3 \
  --input_array=sub_7 \
  --output_array=logits/semantic/BiasAdd \
  --default_ranges_min=0 \
  --default_ranges_max=6 \
  --mean_value=127.5 \
  --std_value=127.5`
I want to run this model on the phone, but the output_array is not the last node in model, and I need to get the float type output data of "logits/semantic/BiasAdd ". Now I use `out_ptr_uint8_t = interpreter->typed_output_tensor<uint8_t>(0);`  to get the uint8_t type output data, can you help me with how to get float type data?

**Comment by tatianashp** [comment]
This question is better asked on [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a bug or feature request. There is also a larger community that reads questions there.

If you think we've misinterpreted a bug, please comment again with a clear explanation, as well as all of the information requested in the [issue template](https://github.com/tensorflow/tensorflow/issues/new). Thanks!
